MobileBuy/sample/README.md

```diff
@@ -0,0 +1,112 @@
+API_KEY=<Your API Key>
+APP_ID=8
+```
+
+
+## Overview
+
+The sample application demonstrates tasks that can be performed using the Android Buy SDK. These tasks have been broken up into a sequential flow:
+
+<ol>
+<li><a href="#collection-list">CollectionListActivity</a></li>
+<li><a href="#product-list">ProductListActivity</a></li>
+<li><a href="#shipping-rate-list">ShippingRateListActivity</a></li>
+<li><a href="#discount">DiscountActivity</a></li>
+<li><a href="#checkout">CheckoutActivity</a></li>
+</ol>
+
+<h3 id="collection-list">CollectionListActivity</h3>
+
+The first activity in the application flow. This activity allows you to browse the list of collections and drill down into a list of products. An item called **All Products** is added to the top of the list for convenience.
+
+<h3 id="product-list">ProductListActivity</h3>
+
+This activity allows you to select a product to purchase from a list of all products in a collection. There are also several additional controls above the list:
+
+* **Demo ProductDetailsActivity** - controls whether selecting a product will display the next Activity in the sample application flow, or launch `ProductDetailsActivity` (a UI component in the SDK).
+
+* **Theme Style** - controls the style of the theme that will appear in the `ProductDetailsActivity` (either `ProductDetailsTheme.Style.LIGHT` or `ProductDetailsTheme.Style.DARK`).
+
+* **Shows Product Image BG** - controls whether the `ProductDetailsActivity` will show a background color behind the product images.
+
+* **Choose Accent Color** - displays a color picker that controls the accent color for the `ProductDetailsActivity` theme.
+
+<h3 id="shipping-rate-list">ShippingRateListActivity</h3>
+
+If the selected product requires shipping, this activity allows the user to select a list of shipping rates. For the sample application, the shipping address has been hardcoded and you will only see the shipping rates applicable to that address.
+
+<h3 id="discount">DiscountActivity</h3>
+
+After a shipping rate is selected, this activity allows the user to add discount codes or gift card codes to the order. It also shows a summary of the order, including the line item price, any discounts or gift cards used, the shipping charge, the taxes, and the total price.
+
+<h3 id="checkout">CheckoutActivity</h3>
+
+The final activity in the app flow. Allows the user to choose between either:
+
+* a **native checkout** - payment info is hardcoded and the checkout is completed within the app, or
+* a **web checkout** - the user enters their payment info and completes the checkout in a web browser.
+
+## Customer Account
+The overflow menu has two items for Customer Accounts, Log In and Orders.
+
+### CustomerLoginActivity
+This activity shows how to sign up a new customer, or log an existing customer in.
+
+### CustomerOrderListActivity
+This activity shows how to retrieve the orders associated with a customer account.
+
+
+## Android Pay
```

Comment:
ah, I see - Test vs. Production.  Ya, we should probably add some more info on that.